lib/cartopy/mpl/geoaxes.py

```diff
@@ -49,7 +49,7 @@
 import cartopy.mpl.patch as cpatch
 from cartopy.mpl.slippy_image_artist import SlippyImageArtist
 from cartopy.vector_transform import vector_scalar_to_grid
-
+from cartopy import geodesic
```

Comment:
There should still be two newlines here for PEP8. Also, I don't know why you import it like this when it gets used as `cartopy.geodesic.Geodesic`.